src/store/rewards/saga.ts

```diff
@@ -38,13 +38,13 @@ export function* fetch(_action) {
     if (result.success) {
       yield put(setMeow(result.response.meow.toString()));
       yield put(setMeowPreviousDay(result.response.meowPreviousDay.toString()));
-
-      yield call(checkNewRewardsLoaded);
     } else {
     }
   } catch (e) {
   } finally {
     yield put(setLoading(false));
+
+    yield call(checkNewRewardsLoaded);
```

Comment:
Why did we move this into the `finally`? If the `fetchRewards` call fails what good does it do us to then see if your rewards changed?

Reply:
wanted to call `checkNewRewardsLoaded` after Loading has been set to `false`. (so we're able to display the rewards tooltip). Otherwise in the `checkNewRewardsLoaded` we're setting `showRewardsTooltip` to `true`, but we're still in "loading state".